src/unix/linux-core.c

```diff
@@ -617,6 +617,11 @@ static uint64_t read_cpufreq(unsigned int cpunum) {
 
 
 static int uv__ifaddr_exclude(struct ifaddrs *ent, int exclude_type) {
+#ifndef HAVE_IFADDRS_H
```

Comment:
Suggestion: wrap this function in its entirety in an #ifdef guard. If `<ifaddrs.h>` doesn't exist, then chances are good `struct ifaddrs` doesn't either.

Reply:
OK. changed.